mappings/net/minecraft/client/gui/IButton.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cwu
+CLASS cwu net/minecraft/client/gui/IButton
```

Comment:
needs a better name

Reply:
We want to avoid using `I` as a prefix if we can. Currently the only I-prefixed class is `IWorld`, and we've been looking to change it too.

Reply:
It's an interface implemented by all button widgets. Any suggestions?

Reply:
It could just be named `Button`.

Reply:
After checking again, it's only implemented by buttons used on the realms screen and provides a method to get the `net.minecraft.realms` proxy class, so I named it `RealmsButton`.